s3/s3_operations.cpp

```diff
@@ -2231,6 +2267,14 @@ namespace irods_s3 {
 
     irods::error s3_notify_operation( irods::plugin_context& _ctx,
         const std::string* str ) {
+        if (is_cacheless_mode(_ctx.prop_map())) {
+
+            // Must update the physical_path in the L1desc[] table for decoupled naming.
+            // In the case of a redirect, this runs on the original connected server and this
+            // is the server that updates the database. This will open update for creates.
```

Comment:
maybe this sentence can be clarified...
"This will open update for creates"

Reply:
The "open" should be "only". But I rewrote this to make it all more clear.